scania-digital-design-system/tegel
# fix(dropdown): update the value attribute of the dropdown when defaulValue is set

**Describe pull-request**  
This PR updates the `value` attribute of the Dropdown when the `default-value` is set. It also adds some basic unit tests for the Dropdown.

**Solving issue**  
Fixes: -

**How to test**  
1. Go to Dropdown
2. Set a `defaultValue`
3. Inspect the DOM and make sure the `value` attribute is set on the dropdown with the default value
4. Select another value manually
5. Inspect the DOM and make sure the `value` attribute is set on the dropdown with the new value

**Suggested test steps**
